backend/curriculum_tracking/api_views.py

```diff
@@ -1045,3 +1033,10 @@ class ContentItemAgileWeightViewSet(viewsets.ModelViewSet):
     serializer_class = serializers.ContentItemAgileWeightSerializer
     filter_backends = [DjangoFilterBackend]
     filterset_fields = ["content_item", "weight"]
+
+
+class CourseRegistrationViewset(viewsets.ModelViewSet):
+    queryset = models.CourseRegistration.objects.all().order_by("user")
```

Comment:
You can add filters here.

```
filterset_fields = ['user','curriculum']
```
This will let you filter by those ids. See my comments on the serializer, it's important that these are ids and not strings.